### DISCUSSION TITLE
Installing NocoDB on TrueNAS [Jun 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2297] [Discord Threads]
> 🙋‍♂️ **cirsten**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi! Personally I don’t have experience on it.    
####    
#### But, on truenas you should have the possibility to create virtual machine trough Docker or a similar software.  
#  
> 🙋‍♂️ **FrancyMak**  
#### And I found this resource in the web that may be useful in your case https://www.truenas.com/community/resources/nocodb-is-the-new-airtable.175/  
#  
> 💁 **o1lab**  
#### <@960663159637688360> : https://github.com/nocodb/nocodb/tree/develop/scripts/setups/TrueNAS  
#  
> 🙋‍♂️ **cirsten**  
#### <@310157751395287040> that setup is for TrueNAS CORE which is a little different that TrueNAS  SCALE, uses different ios (I think) and file system. But i'll give it a go if no other options present  
#  
> 💁 **o1lab**  
#### sure, please send a PR once you have it to work   
####    
#### it will help TrueNAS folks  
#  
> 🙋‍♂️ **cirsten**  
#### that's a bold assumption 🙂 (that I it get it to work)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981121215366307880)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
